services/movie_session.py

```diff
@@ -42,3 +44,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return list(
```

Comment:
or you can do this `list(get_movie_session_by_id(movie_session_id).tickets.values("row", "seat"))`